tests/queries/0_stateless/01926_order_by_desc_limit.sql

```diff
@@ -11,9 +11,11 @@ SETTINGS index_granularity = 1024, index_granularity_bytes = '10Mi';
 INSERT INTO order_by_desc SELECT number, repeat('a', 1024) FROM numbers(1024 * 300);
 OPTIMIZE TABLE order_by_desc FINAL;
 
-SELECT s FROM order_by_desc ORDER BY u DESC LIMIT 10 FORMAT Null;
+SELECT s FROM order_by_desc ORDER BY u DESC LIMIT 10 FORMAT Null
+SETTINGS max_memory_usage = '400M';
```

Comment:
Maybe we should disabled this test with santitizers?

Reply:
The calculated memory usage should not depend on sanitizers,
while sanitizers will use more memory internally.